hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3AEncryptionAlgorithmValidation.java

```diff
@@ -44,21 +44,21 @@ public void testEncryptionAlgorithmSetToDES() throws Throwable {
     assumeEnabled();
     intercept(IOException.class, "Unknown Server Side algorithm DES", () -> {
 
-        Configuration conf = super.createConfiguration();
-        //DES is an invalid encryption algorithm
-        conf.set(Constants.S3_ENCRYPTION_ALGORITHM, "DES");
-        S3AContract contract = (S3AContract) createContract(conf);
-        contract.init();
-        //extract the test FS
-        FileSystem fileSystem = contract.getTestFileSystem();
-        assertNotNull("null filesystem", fileSystem);
-        URI fsURI = fileSystem.getUri();
-        LOG.info("Test filesystem = {} implemented by {}", fsURI, fileSystem);
-        assertEquals("wrong filesystem of " + fsURI,
-            contract.getScheme(), fsURI.getScheme());
-        fileSystem.initialize(fsURI, conf);
-        throw new Exception("Do not reach here");
-      });
+      Configuration conf = super.createConfiguration();
```

Comment:
nit; if we generate that error "unknown server side algorithm", should we change to "unknown s3 encryption algorithm"

Reply:
Seems like we are already returning "Unknown encryption algorithm " exception string in case of an unknown encryption algorithm. This test was just not running and hence, never failed.